controllers/cluster_controller_test.go

```diff
@@ -242,124 +242,6 @@ var _ = Describe(fdbtypes.ProcessClassClusterController, func() {
 			})
 		})
 
-		Context("change the storage servers per disk", func() {
```

Comment:
Definitely agree. I've also wondered whether we should refactor the tests to call reconcile directly instead of relying on the event-based reconciliation.

Reply:
This sounds interesting but I would address this in another PR/issue.

Reply:
> 
> 
> Definitely agree. I've also wondered whether we should refactor the tests to call reconcile directly instead of relying on the event-based reconciliation.

I'm very glad to see this sentiment :) I'm curious what led to this change of heart?

Reply:
I've been thinking about it based on the problems we've had with race conditions in the tests.

Reply:
Ah yes.. I think that was implicit for me in the earlier discussion we had ~ 6 months ago - thank you : this helps me be able to communicate and surface such things.